content/work/timedesigner.md

```diff
@@ -5,18 +5,52 @@ logo: tegy
 client: &client Tegy
 date: 2016-01-01
 end: 2018-07-15
-tags:
-  - Education Sector
 image:
   src: work/timedesigner/grid-desktop.jpg
   alt: School schedule grid
+fullgrid:
+  - img: work/timedesigner/full-grid.jpg
+    alt: a completed scenario with experiences laid out in a grid
+experiencegrid:
+  - img: work/timedesigner/experience-editing.jpg
+    alt: browser inspector showing grid markup and css for a single experience
+emptygridoverlay:
+  - img: work/timedesigner/empty-grid-overlay.jpg
+    alt: Firefox Grid Inspector tool including an overlay of the grid lines
+scenariogrid:
+  - img: work/timedesigner/setting-up-scenario-grid.jpg
+    alt: an empty resource planning grid showing the start and end times
+overlapexperience:
+  - img: work/timedesigner/overlap-experience.jpg
+    alt: a row with overlapping experiences
 people:
   - &furman
     name: Furman Brown
-    # face: furman-brown.jpg
+    face: furman-brown.jpg
     url: https://www.tegy.tools/profile/furman/profile
     title: Founder
     venue: *client
+tags:
+  - _post
+  - Case Study
+  - Education Sector
+sample:
+  desktop:
+    src: timedesigner/timedesigner-desktop.jpg
+    alt: Chicago K-8 School schedule iteration
+  caption: |
+     School scheduling -- from typical schedule
+     to an iteration with expanded planning time
+     for teachers -- built with CSS grid.
```

Comment:
I'm not sure if this wording is intentional or a typo. It might be clearer to me like:

```suggestion
     School scheduling -- from a typical schedule
     to an iteration with expanded planning time
     for teachers -- built with CSS grid.
```

Reply:
I re-read that caption and decided the sentence structure is too convoluted. Changed it to this instead: 
"School scheduling tool -- built with CSS Grid -- allows users to create innovative schedule options."